configs/server/forseti_conf_server.yaml.in

```diff
@@ -215,6 +209,15 @@ scanner:
 
 notifier:
 
+    # Provide connector details to receive violations when found
+    email_connector_config:
+          name: sendgrid
+          auth:
+             api_key: SENDGRID_API_KEY
```

Comment:
Surround the capitalized words with `{ }`, so that they will be populated by the installer.
`{SENDGRID_API_KEY}`

Reply:
Done